geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/operations/RemoveRequestOperationHandler.java

```diff
@@ -35,14 +36,14 @@
 import static org.apache.geode.internal.protocol.ProtocolErrorCode.*;
 
 @Experimental
-public class RemoveRequestOperationHandler
-    implements OperationHandler<RegionAPI.RemoveRequest, RegionAPI.RemoveResponse> {
+public class RemoveRequestOperationHandler implements
+    OperationHandler<RegionAPI.RemoveRequest, RegionAPI.RemoveResponse, ClientProtocol.ErrorResponse> {
   private static final Logger logger = LogManager.getLogger();
 
   @Override
-  public Result<RegionAPI.RemoveResponse> process(SerializationService serializationService,
-      RegionAPI.RemoveRequest request, MessageExecutionContext messageExecutionContext)
-      throws InvalidExecutionContextException {
+  public Result<RegionAPI.RemoveResponse, ClientProtocol.ErrorResponse> process(
```

Comment:
Probably not worth it just to simply out one template parameter, but we could define a:
public ProtobufResult<SuccessResult> extends Result<SuccessResult, ClientProtocol.ErrorResponse>

We could add a similar class extending OperationHandler as well.  This would be consistent with how you added a ProtobufOperationContext.